.cirrus.yml

```diff
@@ -5,14 +5,20 @@
 #   - https://cirrus-ci.org/guide/windows/
 #   - https://hub.docker.com/_/gcc/
 #   - https://hub.docker.com/_/python/
+#   - https://community.codecov.io/t/add-support-of-uploading-from-cirrus-ci-without-token/1028/14
```

Comment:
This link points to a particular comment which doesn't look particularly relevant. Removing the "/14" at the end links to the top of the feature request. This seems like it would make more sense, unless i'm missing something in the comments.